baseapp_chats/graphql/mutations.py

```
@@ -404,6 +384,164 @@ def mutate_and_get_payload(
+                            messages=[_("You can't add this participant to a chatroom")],
+                        )
+                    ]
+                )
+
+        # Resolve and validate every room before any write (all-or-nothing)
+        rooms = []
+        seen_room_pks = set()
+        for room_id in room_ids:
+            try:
+                room = get_obj_from_relay_id(info, room_id)
+            except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)